pd/src/wallet.rs

```diff
@@ -118,7 +131,7 @@ impl ThinWallet for State {
                     .map_err(|_| tonic::Status::unavailable("database error"))
                     .unwrap();
                 for asset in &assets[..] {
-                    tracing::debug!(asset_id = ?hex::encode(&asset.asset_id), asset_denom = ?asset.asset_denom, "sending asset");
+tracing::debug!(asset_id = ?hex::encode(&asset.asset_id), asset_denom = ?asset.asset_denom, "sending asset");
```

Comment:
Not sure if the unindenting was a rebase artifact or something (it doesn't look like this changed?) but, while looking at it, it could be nice to format the asset ID using its `Display` impl.  If the parsing of the asset ID (which I think is raw bytes here?) is done *inside* the `debug!` macro expression (where the `hex::encode` is now), we won't pay any cost unless the tracing filter would actually use the output.

Reply:
I think this was just from running cargo fmt, weird